src/lib/rest/rest.cpp

```diff
@@ -1072,6 +1101,18 @@ static int connectionTreat
     LM_T(LmtUriParams, ("'default' value for notifyFormat (ciP->outFormat == %d)): '%s'", ciP->outFormat, ciP->uriParam[URI_PARAM_NOTIFY_FORMAT].c_str()));
   }
 
+  //
+  // Here, if the incoming request was too big, return error abouyt it
+  //
+  if (ciP->httpHeaders.contentLength > PAYLOAD_MAX_SIZE)
```

Comment:
Let's create an issue. No time for this today, and the rest of the fix is necessary.
Not 100% easy to test this, but it might be a good idea to protect ourselves against
bad usage.

Issue #1451